### DISCUSSION TITLE
What song are in the demo video?

### DISCUSSION THREAD

**Discussion by Mastergamer433** [ivan-hc/AM#27] [Q&A]
So I really like Metal music, big fan. And I like the song in the demo video in the README. What is that song called? I know this does not have to do with the program itself, but I really want that song in my playlist.

**Comment by ivan-hc** [comment]
Song "Start Your Engines" by [Ethan Meixsell](https://www.ethanmeixsell.com/)

These songs are available on YouTube Studio, if you're a content creator you can download them.

A complete playlist is available at https://www.youtube.com/playlist?list=PLKCo4y0JbV2aPETcF1dH6R6yDfpGVbC2p

My favourite one is "Demilitarized Zone" \m/

**Reply by Mastergamer433** [reply]
Thanks so much! Yeah I also liked "Demilitarized Zone", but without have listened to it many times I think "By The Sword" is my favourite, of the songs in that playlist.
